### ISSUE TITLE
模型参数训练问题

### ISSUE THREAD

**Issue by wangyf8848** [PengtaoJiang/L2G#12]
你好，想请教一下，全局网络和局部网络在代码中采用了同一个模型但是属于不同的实例，局部网络的输出调用了.clone().detach()，请问这行代码的作用是什么。两支网络是共享参数吗，如果不是的话局部网络的参数是怎么训练的呢，是和全局网络同时独立的更新参数吗

**Comment by SakaRenko** [comment]
你好，局部网络结果detach()的作用是让其作为没有梯度回传的标签去训练全局网络。
同时，两支网络并不共享参数，因此局部网络和全局网络的训练过程是相互独立的。具体到局部网络来说，就是利用GAP进行图象分类训练来更新参数。
希望能解决你的疑惑
